package.json

```diff
@@ -16,8 +16,7 @@
     "2fa": false,
     "tests": false,
     "cleanup": false,
-    "yarn": false,
-    "contents": "dist"
```

Comment:
This property erroneously sets the name of the package, which violates the publication